src/components/Flows/__tests__/SingleFlowRunHistory.test.tsx

```
@@ -12,41 +13,82 @@
+
+    await act(async () => {
+      renderComponent(defaultFlowRun);
+    });
+
+    await waitFor(() => {
+      expect(errorToast).toHaveBeenCalledWith('Failed to fetch logs', [], expect.any(Object));
+    });
+  });
+
+  it('calls errorToast when fetchLogSummaries fails', async () => {
+    const { httpGet } = require('@/helpers/http');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)